storage/disk/disk.go

```diff
@@ -235,6 +240,166 @@ func (db *Store) NewTransaction(ctx context.Context, params ...storage.Transacti
 	return newTransaction(xid, write, underlying, context, db.pm, db.partitions, db), nil
 }
 
+// Truncate implements the storage.Store interface.
+func (db *Store) Truncate(ctx context.Context, txn storage.Transaction, params storage.TransactionParams, it storage.Iterator) error {
+	newDB, backupDir, err := db.backupAndLoadDB()
+	if err != nil {
+		return wrapError(err)
+	}
+
+	// write new bundle policy and data into the new DB
+	underlying := newDB.NewTransaction(params.Write)
+	xid := atomic.AddUint64(&db.xid, uint64(1))
+	underlyingTxn := newTransaction(xid, params.Write, underlying, params.Context, db.pm, db.partitions, nil)
+
+	var update *storage.Update
+	var lerr error
```

Comment:
[nit] It's a matter of test, I'd go with `err` here. But what does the `l` stand for?

Reply:
I'll update this. `lerr` meant `Loop`(l) `error` (err) => `lerr` 🙃